examples/bin/druid-quickstart.py

```diff
@@ -0,0 +1,337 @@
+            lower_bound_memory_allocation += allocated_memory
+            allocated_services.add(service)
+
+    if lower_bound_memory_allocation > 0:
+        # compute the multiplier again for remaing services
+        memory_weight_sum = 0
+        for service in service_list:
+            if service in allocated_services:
+                continue
+            memory_weight_sum += SERVICE_MEMORY_DISTRIBUTION_WEIGHT.get(service)
+        multiplier = (total_memory - lower_bound_memory_allocation) / memory_weight_sum
+
+    for service in service_list:
+        if service in allocated_services:
+            continue
+        allocated_memory = SERVICE_MEMORY_DISTRIBUTION_WEIGHT.get(service) * multiplier
+        if service in SERVICE_MEMORY_LOWER_BOUND and allocated_memory < SERVICE_MEMORY_LOWER_BOUND.get(service):
+            allocated_memory = SERVICE_MEMORY_LOWER_BOUND.get(service)
+
+        heap_memory = SERVICE_MEMORY_HEAP_PERCENTAGE.get(service) * allocated_memory
+        direct_memory = allocated_memory - heap_memory
+        service_memory_config[service] = build_memory_config_string(int(heap_memory), int(direct_memory))
+
+    print(f'\nMemory distribution for services:')
+    for key, value in service_memory_config.items():
+        print(f'{key}, memory_config: {value}')
+    print('\n')
+
+    return service_memory_config
+
+def build_supervise_script_arguments(service_list, service_memory_config, root_config_path, run_zk):
+    argument_list = []
+
+    argument_list.append("\":verify bin/verify-java\"")
+    argument_list.append("\":verify bin/verify-default-ports\"")
+    argument_list.append("\":notify bin/greet\"")
+    argument_list.append("\":kill-timeout 10\"")
+
+    if run_zk:
+        argument_list.append("\"!p10 zk bin/run-zk conf\"")
+
+    for service in service_list:
+        prefix = ''
+        if service == MIDDLE_MANAGER_SERVICE_NAME:
+            prefix = '!p90 '
+
+        jvm_args = service_memory_config.get(service)
+
+        if jvm_args is None:
+            argument_list.append(f'\"{prefix}{service} bin/run-druid {service} {root_config_path}\"')
+        else:
+            argument_list.append(f'\"{prefix}{service} bin/run-druid {service} {root_config_path} \'{jvm_args}\'\"')
+
+    print('Supervise script command:')
+    for item in argument_list:
+        print(item)
+
+    print('\n')
+
+    return ",".join(argument_list)
+
+def display_help():
+    text = """
+    Usage: start-druid [options]
+
+    where options include:
+       totalMemory=<value>
+            Integer value is supported with 'm' or 'g' suffix.
+            Memory for druid services, if totalMemory is not specified
+            80 percent of system memory is used.
+            Note, if service specific jvm config is present,
+            <totalMemory> shouldn't be specified.
+            <totalMemory> should be greater than equals 3g
```

Comment:
Please try to make the help text a little less verbose. One or two lines for each param should be sufficient.

e.g. for totalMemory

```suggestion
            Total memory for all the processes (services and tasks, if any).
            Default = 80% of total system memory.
            This parameter is ignored if each service already has a jvm.config in the given conf directory.
            e.g. 500m, 4g, 6g
```

Reply:
I have tried to cut it down.